frontend/src/routes/getPagesData.tsx

```diff
@@ -79,7 +84,7 @@ export const getPagesData = (stripePromise: any) =>
       element: <AdminLoginApp />
     },
     {
-      path: "/payment/:payAmount",
+      path: "/payment/:cid/:aid/:price",
```

Comment:
ไหนๆ ก้ไหนๆ แก้เลยได้ไหม ไม่เอา aid, cid

Reply:
in dis